apps/docs/stories/components/OrderTerminals/index.stories.tsx

```diff
@@ -0,0 +1,88 @@
+import type { Meta, StoryObj } from "@storybook/web-components";
+import { withActions } from "@storybook/addon-actions/decorator";
+import { customStoryDecorator, StoryBaseArgs } from "../../utils";
+
+import "@justifi/webcomponents/dist/module/justifi-order-terminals";
+import { ThemeNames } from "../../themes";
+
+type Story = StoryObj;
+
+const storyBaseArgs = new StoryBaseArgs(["business-id", "account-id", "auth-token"]);
+
+const meta: Meta = {
+  title: "Merchant Tools/Order Terminals",
+  component: "justifi-order-terminals",
+  args: {
+    ...storyBaseArgs.args,
+    shipping: false,
+    "submit-button-text": "Order Terminals",
+    Theme: ThemeNames.Light,
+  },
+  argTypes: {
+    ...storyBaseArgs.argTypes,
+    shipping: {
+      description: "Whether should set `order_type` to `boarding_shipping` or `boarding_only`.",
```

Comment:
Not sure I understand this sentence correctly. Would it be correct to say:
"Defines the `order_type`; can be set to  `boarding_shipping` or `boarding_only`."